client/client.go

```diff
@@ -0,0 +1,1662 @@
+	}
+	data, _, err := req.Execute()
+	return data, err
+}
+
+func (client *OpenFgaClient) ListStores(ctx _context.Context) SdkClientListStoresRequest {
+	return SdkClientListStoresRequest{
+		ctx:    ctx,
+		Client: *client,
+	}
+}
+
+// / CreateStore
+type SdkClientCreateStoreRequest struct {
+	ctx    _context.Context
+	Client OpenFgaClient
+
+	body    *ClientCreateStoreRequest
+	options *ClientCreateStoreOptions
+}
+
+type ClientCreateStoreRequest struct {
+	Name string `json:"name"`
+}
+
+type ClientCreateStoreOptions struct {
+	AuthorizationModelId *string `json:"authorization_model_id,omitempty"`
+}
+
+type ClientCreateStoreResponse = openfga.CreateStoreResponse
+
+func (request SdkClientCreateStoreRequest) Options(options ClientCreateStoreOptions) SdkClientCreateStoreRequest {
+	request.options = &options
+	return request
+}
+
+func (request SdkClientCreateStoreRequest) Body(body ClientCreateStoreRequest) SdkClientCreateStoreRequest {
+	request.body = &body
+	return request
+}
+
+func (request SdkClientCreateStoreRequest) Execute() (ClientCreateStoreResponse, error) {
+	return request.Client.CreateStoreExecute(request)
+}
+
+func (client *OpenFgaClient) CreateStoreExecute(request SdkClientCreateStoreRequest) (ClientCreateStoreResponse, error) {
+	data, _, err := client.OpenFgaApi.CreateStore(request.ctx).Body(openfga.CreateStoreRequest{
+		Name: request.body.Name,
+	}).Execute()
+	return data, err
+}
+
+func (client *OpenFgaClient) CreateStore(ctx _context.Context) SdkClientCreateStoreRequest {
+	return SdkClientCreateStoreRequest{
+		Client: *client,
+		ctx:    ctx,
+	}
+}
+
+// / GetStore
+type SdkClientGetStoreRequest struct {
+	ctx    _context.Context
+	Client OpenFgaClient
+
+	options *ClientGetStoreOptions
+}
+
+type ClientGetStoreOptions struct {
+}
+
+type ClientGetStoreResponse = openfga.GetStoreResponse
+
+func (request SdkClientGetStoreRequest) Options(options ClientGetStoreOptions) SdkClientGetStoreRequest {
+	request.options = &options
+	return request
+}
+
+func (request SdkClientGetStoreRequest) Execute() (ClientGetStoreResponse, error) {
+	return request.Client.GetStoreExecute(request)
+}
+
+func (client *OpenFgaClient) GetStoreExecute(request SdkClientGetStoreRequest) (ClientGetStoreResponse, error) {
+	data, _, err := client.OpenFgaApi.GetStore(request.ctx).Execute()
+	return data, err
+}
+
+func (client *OpenFgaClient) GetStore(ctx _context.Context) SdkClientGetStoreRequest {
+	return SdkClientGetStoreRequest{
+		Client: *client,
+		ctx:    ctx,
+	}
+}
+
+// / DeleteStore
+type SdkClientDeleteStoreRequest struct {
+	ctx    _context.Context
+	Client OpenFgaClient
+
+	options *ClientDeleteStoreOptions
+}
+type ClientDeleteStoreOptions struct {
+	AuthorizationModelId *string `json:"authorization_model_id,omitempty"`
+}
+
+type ClientDeleteStoreResponse struct{}
+
+func (request SdkClientDeleteStoreRequest) Options(options ClientDeleteStoreOptions) SdkClientDeleteStoreRequest {
+	request.options = &options
+	return request
+}
```

Comment:
remove

Reply:
These will remain to be consistent - it's optional, and will house overrides for the request